cnf-certification-test/operator/suite.go

```diff
@@ -61,3 +67,34 @@ func testOperatorInstallationPhaseSucceeded(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("Found %d CSVs whose phase is not %s.", n, v1alpha1.CSVPhaseSucceeded))
 	}
 }
+
+func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
+	badCsvs := []string{}
+	if len(env.Csvs) == 0 {
+		ginkgo.Skip("No CSVs to perform test, skipping.")
+	}
+
+	for _, csv := range env.Csvs {
+		clusterPermissions := csv.Spec.InstallStrategy.StrategySpec.ClusterPermissions
+		if len(clusterPermissions) == 0 {
+			logrus.Debugf("No clusterPermissions found in csv %s (ns %s)", csv.Name, csv.Namespace)
+			continue
+		}
+
+	LOOP:
```

Comment:
I'm torn about using labels/goto functions.  Obviously it works but I don't know if I like it...

Reply:
I think this whole thing needs to be broken down to reduce complexity.  Also allows for more granule unit testing to be added.

Reply:
I'll remove the loop break based on David's suggestion.